### PR TITLE
Added archived flag to projects table.

### LINKED ISSUE #149: Add "archived" flag to project info
We need to add an "archived" flag for projects to DB.  This would be for completed projects

### PR DESCRIPTION
**Pull request by OBCENEIKON**
Fixes #149

### CODE REVIEW

**Comment by chriskacerguis** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54d4fa64695680762a0024c5/compare/feature/JEL-149-Add_archived_flag_to_projects">analyzed this pull request</a>.

File: db/migrate/20150225233218_add_archived_to_projects.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddArchivedToProjects < ActiveRecord::Migration
+  def change
+    add_column :projects, :archived, :boolean, default: false, null: false
```
Comment (stackus) [inline db/migrate/20150225233218_add_archived_to_projects.rb]: If this was a datetime instead you could capture two bits of information with a single column.

A null would indicate it is not archived, anything else is archived. Bonus is knowing when it was archived.

Also consider requesting projects and choosing to include or exclude archived results. In the very least you may also want to add a scope to include/exclude archived projects.

**Comment by chriskacerguis** [comment]
@OBCENEIKON thoughts?

**Comment by OBCENEIKON** [comment]
I like it, never even thought about doing it like that. I’ll make that change now.

Thanks,
Jerimiah Milton

 [cid:814279EB-8844-468A-8468-EB7C762F305D]
Systems Engineer – Strategic Innovation Group
254.493.9153<tel:254.493.9153>
Milton_Jerimiah@bah.com<mailto:Milton_Jerimiah@bah.com>

**Comment by chriskacerguis** [comment]
:+1:

File: db/schema.rb [hunk 2]
```diff
@@ -11,11 +11,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150225203330) do
+ActiveRecord::Schema.define(version: 20150227173037) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
+  enable_extension "tablefunc"
```
Comment (stackus) [inline db/schema.rb]: This snuck in. Any info on what this was added?

File: db/schema.rb [hunk 3]
```diff
@@ -295,10 +296,23 @@
     t.decimal  "spent",                  precision: 12, scale: 2, default: 0.0
     t.integer  "status",                                          default: 0
     t.integer  "approval",                                        default: 0
+    t.datetime "archived"
   end
 
+  add_index "projects", ["archived"], name: "index_projects_on_archived", using: :btree
   add_index "projects", ["deleted_at"], name: "index_projects_on_deleted_at", using: :btree
 
+  create_table "provision_derivations", force: true do |t|
```
Comment (stackus) [inline db/schema.rb]: This table was also added but there is no migration for it.

**Comment by OBCENEIKON** [comment]
tablefunc and provision_derivations got in by accident, some of my changes from a different branch snuck in somehow.

**Pull request merged**